src/content/docs/developer-tools/kinde-api/add-a-m2m-application-for-api-access.mdx

```diff
@@ -18,6 +18,10 @@ Machine to machine applications facilitate token exchange between systems, appli
 
 For example, you need a machine to machine application for connecting to [Kinde’s Management API](https://kinde.com/api/docs/#kinde-management-api). See below for instructions.
 
+## About scopes and the Kinde Management API
```

Comment:
It won't let me add a comment on line 3 - but I'm wondering if the doc title should more closely resemble the slug? `Add a machine to machine application for Kinde management API access`?